### PR TITLE
Added about page

### PR DESCRIPTION
**Pull request by Depechie**
### Description of Change ###

Added an about page to the showcase app.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

File: XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            if (SelectedContributor is null)
+                return;
+
+            await Launcher.OpenAsync(SelectedContributor.HtmlUrl);
+            SelectedContributor = null;
+        });
+
+        public async Task OnAppearing()
+        {
+            if (Contributors == null)
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs:42]: **guard** approach is preferable from the clean code point of view :)

```
if (Contributors != null)
    return
```

File: XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            {
+                var contributors = await gitHubClient.Repository.GetAllContributors("xamarin", "XamarinCommunityToolkit");
+                if (contributors != null)
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs:45]:
```
if (contributors == null)
    return;
```

File: XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs [hunk 3]
```diff
@@ -29,5 +29,10 @@ Page GetPage(WelcomeSectionId id)
             page.Title = id.GetTitle();
             return page;
         }
+
+        async void OnAboutClicked(System.Object sender, System.EventArgs e)
+        {
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs:34]: no need in bracers )

**Comment by AndreiMisiukevich** [comment]
Probably we should add ActivityIndicator during content loading and make avatars bigger

image

File: XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs (hunk 3 above)
Comment (Depechie) [reply XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs:34]: You mean using the new 1 line method call notation? '=>'

**Comment by Depechie** [comment]
Made the avatars a bit bigger and introduced an empty view that shows info about data being loaded ( instead of using a loading indicator, we can always add one if needed ).

File: XamarinCommunityToolkitSample/Pages/AboutPage.xaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+             x:Class="XamarinCommunityToolkitSample.Pages.AboutPage"
+             Title="About">
+
+    <ContentPage.BindingContext>
+        <viewmodels:AboutViewModel />
+    </ContentPage.BindingContext>
+
+    <ContentPage.Content>
+        <StackLayout>
+            <Label Text="Thank you XamarinCommunityToolkit contributers"
```
Comment (sthewissen) [inline XamarinCommunityToolkitSample/Pages/AboutPage.xaml:17]: Should be "Contribut**o**rs"

Comment (Depechie) [reply XamarinCommunityToolkitSample/Pages/AboutPage.xaml:17]: oh oooooh :D

File: XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs (hunk 3 above)
Comment (AndreiMisiukevich) [reply XamarinCommunityToolkitSample/Pages/WelcomePage.xaml.cs]: Yeah

**Pull request merged**